### ISSUE TITLE
Add Backward Smoothing

### ISSUE THREAD

**Issue by jinghuichen** [fra31/auto-attack#26]
**Paper**: Efficient Robust Training via Backward Smoothing https://arxiv.org/abs/2010.01278

**Venue**: {if applicable, the venue where the paper appeared}

**Dataset and threat model**: CIFAR-10/CIFAR100, Linf, 8/255

**Code**: https://github.com/jinghuichen/AutoAttackEval

**Pre-trained model**:  https://drive.google.com/file/d/1lvMa2rbMrIVkAqsyrs_YXLBhewZBfdkP/view?usp=sharing (CIFAR10)
https://drive.google.com/file/d/1xNhK4w5ZuUSfbD_WR4xFKTprojaVux1A/view?usp=sharing (CIFAR100)

**Log file**: {link to log file of the evaluation}

**Additional data**: no

**Clean and robust accuracy**: CIFAR10 clean 85.32 robust 54.94 CIFAR100 clean 62.15 robust 31.92

**Architecture**: {wideresnet-34-10}

**Description of the model/defense**: Efficient robust training via backward smoothing

Thanks

**Comment by fra31** [comment]
Hi,

thanks for the submission! I ran the evaluation with Linf-bound `eps=8/255` and got

CIFAR-10
clean accuracy: 85.32%
robust accuracy 51.12%

CIFAR-100
clean accuracy: 62.15%
robust accuracy 26.94%

which seem to me in line with what reported in the paper. If this is the case, I'd be happy to add them!

**Comment by jinghuichen** [comment]
The numbers are correct. Thank you very much!

**Comment by fra31** [comment]
Added, thanks again for the submissions!
